autoswitch_virtualenv.plugin.zsh

```diff
@@ -315,7 +315,9 @@ function mkvenv()
         if [[ -f "$AUTOSWITCH_FILE" ]]; then
             printf "$AUTOSWITCH_FILE file already exists. If this is a mistake use the rmvenv command\n"
         else
-            local venv_name="$(basename $PWD)"
+            # If the mkvenv command is executed in '/home/user/my-project', the name of the virtualenv
+            # will be 'my-project-wyUf‘ where the last 4 characters are random letters of mixed case
+            local venv_name="$(basename $PWD)-$(base64 </dev/urandom | tr -dc 'a-zA-Z' | head -c4)"
```

Comment:
I think its pretty standard to have `pwgen` installed on Linux and MacOSX,

so this can be

```shell
local venv_name=$(basename $PWD)-$(pwgen 8 1)"
```